packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -133,7 +133,7 @@ export class NodeHttpRequestResponseHandler extends NodeHttpStreamsHandler {
           case 405: return of({ ...error, body: 'Method Not Allowed' });
           case 500: return of({ ...error, body: 'Internal Server Error' });
           // If no cases match, return a default error message.
-          default: return error.status < 600 && error.status >= 400 ?  of({ ...error, body: 'An Unexpected Error Occured' }) : of({ body: 'An Unexpected Error Occured', headers: error.headers, status: 500 });
+          default: return of({ ...error, body: 'Internal Server Error', status: 500 });
```

Comment:
This is not what I meant. The entire part that is now the error handler should no longer be here.

Reply:
> Also, I would prefer a test of the errorhandler with a config file, to demonstrate that it works in combination with other handlers (especially the CORS handler).

OK. So that would require like a small integration setup, right?